coral-dbt/src/main/resources/macros/coral_macros/default/utils/generate_incremental_code.sql

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+{% macro generate_incremental_code(coral_response, table_names, output_table) %}
```

Comment:
I feel calling it 'script' instead of 'code' reflects what this macro does. can you try and rename this to something like - 'generate_incremental_script_for_adapter / execution'?

Also, please add comments on the macros.